src/utils/tagged-template-literal.js

```diff
@@ -141,6 +141,17 @@ const interleave = (quasis, expressions, absolutePath) => {
        */
       substitute = '$dummyValue'
     }
+    // Make sure substituted by same count of lines
+    const intentCount = prevText.split('\n').pop().length
+    const intent = new Array(intentCount).fill(' ').join('')
```

Comment:
Hmm, I think we might also be making similar mistakes like this other places in the code (I don't quite remember if we handle it perfectly everywhere, such as in our indentation fixers) but would this break some indentation Stylelint rules if someone was using tabs? Adding a test that tests that would definitely be great whether this is a breaking test or not :)

Reply:
Thanks very much for correcting me! A better way may be selecting all left spaces from the last line, i.e. `prevText.split('\n').pop().replace(/^(\s*).*$/, '$1')`.